### PR TITLE
[fix] Fix rendering issue on Select form search suggestions list

### PR DESCRIPTION
**Pull request by arsengit**
- Fixed select form search suggestions list

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #1384

File: aim/web/ui/src/utils/app/getParamsSuggestions.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * @returns memoized params suggestions from projectsData
+ */
+
+export function getParamsSuggestions(data: IProjectsModelState) {
+  let list: string[] = [];
+  if (data?.params) {
+    Object.keys(data?.params).forEach((option: any) => {
+      if (option) {
+        list.push(`run.${option}`);
+        if (data.params) {
```
Comment (Hamik25) [inline aim/web/ui/src/utils/app/getParamsSuggestions.ts:13]: I think here can be used an operator and one if statement with two conditions. data?.params[option] can be changed to data.params[option] if data hasn't param property or param prop is falsely value this scenario will not be executed.

**Review by Hamik25 — APPROVED** [review]

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Review by rubenaprikyan — CHANGES_REQUESTED** [review]
Update CHANGE LOG to  **Display `None` values  in group config column.**

**Pull request merged**